srcpkgs/zeromq/template

```diff
@@ -1,25 +1,24 @@
 # Template file for 'zeromq'
 pkgname=zeromq
-version=4.3.0
+version=4.3.1
 revision=1
 build_style=gnu-configure
 configure_args="--with-libsodium"
 hostmakedepends="asciidoc pkg-config xmlto"
 makedepends="libsodium-devel"
-short_desc="The Intelligent Transport Layer"
+short_desc="Intelligent Transport Layer"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="LGPL-3.0-or-later"
 homepage="http://www.zeromq.org"
 distfiles="https://github.com/zeromq/libzmq/releases/download/v${version}/zeromq-${version}.tar.gz"
-checksum=8e9c3af6dc5a8540b356697081303be392ade3f014615028b3c896d0148397fd
+checksum=bcbabe1e2c7d0eec4ed612e10b94b112dd5f06fcefa994a0c79a45d835cd21eb
 
 zeromq-devel_package() {
 	depends="${sourcepkg}-${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
-		vmove usr/share/man/man3
```

Comment:
Why? I don't see a changelog item in zmq for this

Reply:
If you are talking about the man page, I don’t understand why man page is in devel package

Reply:
Ah didn’t know that, makes sense now

Reply:
thank you.